security-policies/bundle/compliance/policy/azure/storage_account/ensure_service_log.rego

```diff
@@ -0,0 +1,14 @@
+package compliance.policy.azure.storage_account.ensure_service_log
+
+import future.keywords.if
+import future.keywords.in
+
+service_diagnostic_settings_log_enabled(serviceDiagnosticSettings, categories) if {
```

Comment:
Why receive categories by parameter if the are always the same `["StorageRead", "StorageWrite", "StorageDelete"]`?

Reply:
It felt weird to hardcode the categories inside the function. It would make the function kind of rule-specific. wdyt?

Reply:
I think it's fine to make it rule specific, like it always check if rwd are enabled

Reply:
Sure, refactored the function a bit.